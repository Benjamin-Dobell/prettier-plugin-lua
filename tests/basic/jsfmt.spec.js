run_spec(__dirname, ["lua51"]);
