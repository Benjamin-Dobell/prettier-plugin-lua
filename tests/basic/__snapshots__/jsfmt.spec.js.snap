// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic.lua - lua51-verify: basic.lua 1`] = `
local foo, bar = 4

someglobal = 5, 6, 7

local ok = false
function bar()
    print("hi")
    print()
end

tbl[a] = 3
tbl.a = 3
tbl:a()
tbl["a"][4] = 3

function MT:bla()
end

local tbl = {
    1,
    2,
    3,
    a = 4,
    a = 6,
    ["bla"] = 69,
    [b] = {
        [32] = 5,
    },
    [{ a = 5 }] = 6,
  	4, 5, 6
}

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
local foo, bar = 4

someglobal = 5, 6, 7

local ok = false
function bar()
	print("hi")
	print()
end

tbl[a] = 3
tbl.a = 3
tbl:a()
tbl["a"][4] = 3

function MT:bla()

end

local tbl = {
	1,
	2,
	3,
	a = 4,
	a = 6,
	["bla"] = 69,
	[b] = {
		[32] = 5
	},
	[{
		a = 5
	}] = 6,
	4,
	5,
	6
}
`;
